eqcorrscan/core/match_filter/template.py

```diff
@@ -75,7 +75,7 @@ class Template(object):
     def __init__(self, name=None, st=None, lowcut=None, highcut=None,
                  samp_rate=None, filt_order=None, process_length=None,
                  prepick=None, event=None):
-        name_regex = re.compile(r"^[a-z_0-9]+$")
+        name_regex = re.compile(r"^[-A-Za-z_0-9]+$")
```

Comment:
Caps will cause problems on some systems,  see note in issue. Just add the dash.

Reply:
I'm sorry for it.
I didn't understand this last night.